ui/admin/setup-edit.php

```diff
@@ -813,7 +813,7 @@
                             </div>
                             <div id="publishing-action">
                                 <img class="waiting" src="<?php echo esc_url( admin_url( 'images/wpspin_light.gif' ) ); ?>" alt="" />
-                                <button class="button-primary" type="submit">Save Pod</button>
+                                <button class="button-primary" type="submit"><?php _e( 'Save Pod', 'pods' ); ?></button>
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.